src/client/ui/App.js

```diff
@@ -201,6 +201,7 @@ class App extends Component {
   };
 
   componentDidMount() {
+    this.props.editor.project.watch();
```

Comment:
Note: This line was removed in a previous PR and caused scenes to not upload properly.